semantic_conventions/trace/messaging.yaml

```
@@ -1,4 +1,40 @@
 groups:
+  - id: messaging.message
+    prefix: messaging
+    type: span
+    brief: 'Semantic convention describing per-message attributes populated on messaging spans or links.'
+    attributes:
+      - id: destination.name
+        type: string
+        brief: 'The message destination name'
+        note: |
+          Destination name SHOULD uniquely identify specific queue, topic, or other entity within broker. If
+          broker does not have such notion, destination name SHOULD uniquely identify broker.
+
+          If destination name is the same for all messages being published, 
+          it MUST be set on corresponding publish, receive, process or other span.
```

Do we need to have `or other span` here? If so, maybe something like: 

```suggestion
          it MUST be set on the corresponding publish, receive and process span. It MAY also be set on other spans related to any of aforementioned operations.
```

---

removed the sentence since we decided to address attribute placement (on spans or links) in a separate PR.